src/main/java/at/hannibal2/skyhanni/api/SkillAPI.kt

```diff
@@ -0,0 +1,445 @@
+            var levelMatcher = skillTabPattern.matcher(line)
+            if (levelMatcher.matches()) {
+                tablistLevel = levelMatcher.group("level").toInt()
+                if (levelMatcher.group("type").lowercase() != activeSkill?.lowercaseName) tablistLevel = 0
+            } else {
+                levelMatcher = maxSkillTabPattern.matcher(line)
+                if (levelMatcher.matches()) {
+                    tablistLevel = levelMatcher.group("level").toInt()
+                    if (levelMatcher.group("type").lowercase() != activeSkill?.lowercaseName) tablistLevel = 0
+                }
+            }
+        }
+        val existingLevel = getSkillInfo(skillType) ?: SkillInfo()
+        val xpPercentageS = matcher.group("progress").replace(",", "")
+        val xpPercentage = xpPercentageS.toFloatOrNull() ?: return
+        val levelingArray = levelArray()
+        val levelXp = calculateLevelXp(levelingArray, existingLevel.level - 1)
+        val nextLevelDiff = levelingArray[tablistLevel]?.asDouble ?: 7_600_000.0
+        val nextLevelProgress = nextLevelDiff * xpPercentage / 100
+        val totalXp = levelXp + nextLevelProgress
+        val (_, currentOverflow, currentMaxOverflow, totalOverflow) = getSkillInfo(tablistLevel, nextLevelProgress.toLong(), nextLevelDiff.toLong(), totalXp.toLong())
+        existingLevel.apply {
+            this.totalXp = totalXp.toLong()
+            this.currentXp = nextLevelProgress.toLong()
+            this.currentXpMax = nextLevelDiff.toLong()
+            this.level = tablistLevel
+
+            this.overflowTotalXp = totalOverflow
+            this.overflowCurrentXp = currentOverflow
+            this.overflowCurrentXpMax = currentMaxOverflow
+            this.overflowLevel = tablistLevel
+
+            this.lastGain = matcher.group("gained")
+        }
+        storage?.set(skillType, existingLevel)
+    }
+
+    private fun handleSkillPatternMultiplier(matcher: Matcher, skillType: SkillType, skillInfo: SkillInfo) {
+        val currentXp = matcher.group("current").formatNumber()
+        val maxXp = matcher.group("needed").formatNumber()
+        val level = getLevelExact(maxXp)
+        val levelingArray = levelArray()
+        val levelXp = calculateLevelXp(levelingArray, level - 1).toLong() + currentXp
+        val (currentLevel, currentOverflow, currentMaxOverflow, totalOverflow) = getSkillInfo(level, currentXp, maxXp, levelXp)
+        skillInfo.apply {
+            this.overflowCurrentXp = currentOverflow
+            this.overflowCurrentXpMax = currentMaxOverflow
+            this.overflowTotalXp = totalOverflow
+            this.overflowLevel = currentLevel
+
+            this.currentXp = currentXp
+            this.currentXpMax = maxXp
+            this.totalXp = levelXp
+            this.level = level
+
+            this.lastGain = matcher.group("gained")
+        }
+        storage?.set(skillType, skillInfo)
+    }
+
+    fun onCommand(it: Array<String>) {
```

Comment:
Remove the duplicate [skyhanni] prefix. Use a different color for the description
![grafik](https://github.com/hannibal002/SkyHanni/assets/24389977/6704ff37-3e31-4a5f-9ff5-8ffbb485c938)